internal/printer/printer_test.go

```diff
@@ -96,6 +96,16 @@ import VueComponent from '../components/Vue.vue';
   </body></html>`,
 			},
 		},
+		{
+			name:   "ternary",
+			source: `<body>{false ? <div>#f</div> : <div>#t</div>}</body>`,
+			want: want{
+				imports:     "",
+				frontmatter: []string{},
+				styles:      []string{},
+				code:        "<html><head></head><body>${false ? $$render`<div>#f</div>` : $$render`<div>#t</div>`}</body></html>",
```

Comment:
This was what it was rendering before:

```
${false ? $$render`<div>#f</div>` : <div>#t</div>`}
```

Notice the `$$render` and backtick are missing from the right-hand side